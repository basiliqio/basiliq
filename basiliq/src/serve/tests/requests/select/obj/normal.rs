use super::*;

crate::run_test_request!(
    get,
    Method::GET,
    format!("/public__peoples/{}", PEOPLES_IDS[0]),
    200
);
crate::run_test_request!(
    get_unknown,
    Method::GET,
    format!(
        "/public__peoples/{}",
        "3d046782-2c28-4b68-b148-f4b5afcd0984"
    ),
    404
);
