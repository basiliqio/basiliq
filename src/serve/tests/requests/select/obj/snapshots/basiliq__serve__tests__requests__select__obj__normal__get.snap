---
source: basiliq/src/serve/tests/requests/select/obj/normal.rs
expression: res

---
{
  "jsonapi": {
    "version": "1.0"
  },
  "data": {
    "type": "public__peoples",
    "id": "1649b1e9-8a5f-4f52-b331-c07ce3bccc6f",
    "attributes": {
      "age": 22,
      "first-name": "Francis",
      "gender": "M",
      "last-name": "Le Roy",
      "twitter": null
    }
  }
}
